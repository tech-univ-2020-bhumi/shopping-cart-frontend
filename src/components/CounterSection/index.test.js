import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import Counter from './index';


describe('the Counter component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Counter />);

    expect(asFragment()).toMatchSnapshot();
  });
});
