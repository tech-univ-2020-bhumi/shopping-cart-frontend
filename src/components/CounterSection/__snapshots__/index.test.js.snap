// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Counter component should render correctly 1`] = `
<DocumentFragment>
  <div
    data-testid="counter-div"
  >
    <button
      data-testid="decrement"
      type="button"
    >
      -
    </button>
    <button
      data-testid="increment"
      type="button"
    >
      +
    </button>
  </div>
</DocumentFragment>
`;
